browser/src/control/Control.Tabs.js

```diff
@@ -283,17 +281,27 @@ L.Control.Tabs = L.Control.extend({
 		this._map.sendUnoCommand('.uno:Move?Copy:bool=false&UseCurrentDocument:bool=true&Index=' + newIndex);
 	},
 
+	_moveOrCopySheet: function () {
+		var contextMenuTab = this._tabForContextMenu;
+		this._map.sendUnoCommand('.uno:Move?FromContextMenu:bool=true&MoveOrCopySheetDialog:bool=true&ContextMenuIndex=' + contextMenuTab);
+	},
+
+	_moveSheetLR: function (contextMenuTab, contextMenuTargetTab) {
+		if (contextMenuTab !== undefined && contextMenuTab >= 0)
+			this._map.sendUnoCommand('.uno:Move?Copy:bool=false&UseCurrentDocument:bool=true&FromContextMenu:bool=true&ContextMenuIndex=' + contextMenuTab + '&ContextMenuTargetIndex=' + contextMenuTargetTab);
+	},
+
 	_moveSheetLeft: function () {
-		var targetIndex = this._map._docLayer._partNames.indexOf(this._map._docLayer._partNames[this._map._docLayer._selectedPart]);
 		//core handles sheet with 1 base indexing
 		// 0 index means last sheet
-		if (targetIndex <= 0) return;
-		this._moveSheet(targetIndex);
+		var contextMenuTab = this._tabForContextMenu;
+		if (contextMenuTab <= 0) return;
+		this._moveSheetLR(contextMenuTab, contextMenuTab);
```

Comment:
I don't understand how it works.
Current value and target are the same?

Reply:
Actually it should be: `this._moveSheetLR(contextMenuTab, contextMenuTab - 1);`
But, decreasing the `contextMenuTab` by `1` is made in the core side here: https://gerrit.libreoffice.org/c/core/+/158078/5/sc/source/ui/view/tabvwshf.cxx#514
So, I did not used `contextMenuTab - 1` in the online side.

Seems like this is confusing. I can do the decreasing here, on the online side, if needed?

Reply:
Yes, please do that so it will be obvious how to use the uno command